guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -258,13 +258,15 @@ The same guide contains information on configuring external authentication sourc
 === Additional Tasks
 
 This section provides a short overview of selected {Project} capabilities that can be used for automating certain tasks or extending the core usage of {Project}:
-
+////
```

Comment:
I'm generally not a fan of commenting something. We have git so we can revert if needed. Who knows what may be written then.

Reply:
Ok, I'm removing it.